app/services/rad_common/filter_defaulting.rb

```diff
@@ -0,0 +1,45 @@
+module RadCommon
+  module FilterDefaulting
```

Comment:
for non-class modules, my preference is to not create a separate file simply to break it up

Reply:
Yeah I was kind of back and forth on this pattern too. My main issue was that I was stuffing too much logic into a single class and making it concerned with multiple things which causes too much complexity. I'm going to try to quick refactor and break these out into classes instead. I am putting in a separate branch so that if causes too much trouble we can abort easily